HW04/ivan_klimov/lib/mentor.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class Mentor < Person
+  def add_homework(title:, description:, student:)
+    hw = Homework.new(title: title, description: description, student: student)
```

Comment:
add initializer pls

Reply:
I have a parent class Person, with initializer. So child classes Mentor and Student are inherited from it